.github/workflows/weekly.yml

```diff
@@ -0,0 +1,47 @@
+name: Weekly
+
+on:
+  schedule:
+    - cron: '0 0 * * 0'
```

Comment:
can you do it on a weekday? I am subscribed to everything going on in PayPal and would hate to see this on the weekend ;)